openfisca_us/tests/policy/baseline/irs/misc_tax_sim_test.yaml

```diff
@@ -0,0 +1,54 @@
+- name: Tax unit with id 6
```

Comment:
Good to know Martin's full test passes, but I'd suggest pulling out the CDCC component of this into an integration test in the normal folder structure, using our conventions like naming people as `head`/`spouse`/`child1`/etc., underscore thousands separators, removing extraneous testing variables etc.

Or if we want a set of ad hoc integration tests for multiple variables, could you suggest a structure to fit it in?

Reply:
Yeah thanks - did both